cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+""" Field validation for properties table.
+"""
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pod
+from cfme.containers.route import Route, list_tbl as list_tbl_route
+from cfme.containers.project import Project, list_tbl as list_tbl_proj
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+""" Polarion test case CMP-9911.
+"""
+
+
+@pytest.mark.parametrize('rel',
+                         ['name',
+                          'phase',
+                          'creation_timestamp',
+                          'resource_version',
+                          'restart_policy',
+                          'dns_policy',
+                          'ip_address'
+                          ])
+def test_pods_properties_rel(provider, rel):
+    sel.force_navigate('containers_pods')
+    ui_pods = [r.name.text for r in list_tbl_pod.rows()]
+    mgmt_objs = provider.mgmt.list_container_group()  # run only if table is not empty
+
+    if ui_pods:
+        # verify that mgmt pods exist in ui listed pods
+        assert set(ui_pods).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_pods:
+        obj = Pod(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```

Comment:
This assert is a tautology. the if will let you in only when the field_content is not empty (empty string is false) and then you are checking if the string is empty again, therefore this assert can never be false. :exclamation: